x-pack/plugins/enterprise_search/server/lib/enterprise_search_request_handler.ts

```diff
@@ -69,7 +69,7 @@ export class EnterpriseSearchRequestHandler {
         const queryString = !this.isEmptyObj(queryParams)
           ? `?${querystring.stringify(queryParams)}`
           : '';
-        const url = encodeURI(this.enterpriseSearchUrl + path + queryString);
+        const url = this.enterpriseSearchUrl + path + queryString;
```

Comment:
Great catch! We probably still want to encodeURI paths though - what do you think about this?

```suggestion
        const url = encodeURI(this.enterpriseSearchUrl + path) + queryString;
```

Reply:
Also want to add that I don't feel super strongly about it personally, but Larry or someone from the security team left that comment as review on our original MVP PR that we should be `encodeURI`ing, hence the extra paranoia here

Reply:
This makes sense. Good call.